holoviews/operation/element.py

```diff
@@ -486,6 +486,12 @@ class histogram(Operation):
     bin_range = param.NumericTuple(default=None, length=2,  doc="""
       Specifies the range within which to compute the bins.""")
 
+    bins = param.ClassSelector(default=None, class_=(np.ndarray, list), doc="""
```

Comment:
What about tuples or pandas series? Do we want to support lots of different types or force a single type?

Reply:
I'd be okay adding tuples I suppose, although that doesn't seem particularly common.

Reply:
We should use a known set of types for 'list-like' types. Could you open an issue about this? Doesn't need to be fixed in this PR.

Reply:
After considering this I'm sure this makes sense tbh. It's simply too dependent on the particular use case. In many cases ndarray and lists might be used interchangeably but often supporting arrays doesn't really make sense. Then there's lists and tuples, which again can be used interchangeably in some cases but are treated completely differently in others.

I've now added tuples here, but I don't think that generalizes in any real sense.

Reply:
You're *not* sure it makes sense?

Reply:
Right, fixed.

Reply:
My (not particularly strongly held) opinion is that it should only support lists: ``ndarray`` depends on numpy (not a built-in) and as Philipp points out we have cases where we treat tuples and lists differently. For parameters we would often enforce lists with ``param.List`` instead of a class selector which is what would have probably made more sense here.

Reply:
Anyway, this is a minor point which we can decide on later and fix in a subsequent PR if we want to. I'll go ahead and merge.